vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -902,6 +902,7 @@
     backup: yes
     src: server.yml
     dest: /home/arkcase/.arkcase/acm/acm-config-server-repo/arkcase-server.yaml
+  ignore_errors: true
```

Comment:
What kind of errors are we ignoring here?  Generally I prefer a specific `failed_when` clause, to specify the exact errors that should not cause the task to fail.